content/tutorial/tutorial.md

```diff
@@ -12,185 +12,189 @@ redirect_from:
-* [Overview](#overview) will teach you **the fundamentals** of React: components, props, and state.
-* [Completing the Game](#completing-the-game) will teach you **the most common techniques** in React development.
-* [Adding Time Travel](#adding-time-travel) will give you **a deeper insight** into the unique strengths of React.
+* [ट्यूटोरियल के लिए सेटअप](#setup-for-the-tutorial) आपको ट्यूटोरियल का पालन करने के लिए **शुरुआत** देगा।
+* [ओवरव्यू](#overview) आपको React की **बुनियादी बातें**: कौम्पोनॅन्ट्स, props और स्टेट सिखाएगा।
+* [गेम को पूरा करना](#completing-the-game) आपको React विकास में **सबसे आम तकनीकें** सिखाएगा।
+* [टाइम ट्रैवल को गेम में डालना](#adding-time-travel) से आपको React की अद्वितीय शक्तियों के बारे में गहराई से जानकारी मिलेगी।
 
-You don't have to complete all of the sections at once to get the value out of this tutorial. Try to get as far as you can -- even if it's one or two sections.
+इस ट्यूटोरियल से मूल्य प्राप्त करने के लिए आपको एक बार में सभी वर्गों को पूरा करने की आवश्यकता नहीं है।
+जहां तक ​​हो सके जाने की कोशिश करें -- भले ही यह एक या दो सेक्शन हो।
 
-It's fine to copy and paste code as you're following along the tutorial, but we recommend to type it by hand. This will help you develop a muscle memory and a stronger understanding.
+ट्यूटोरियल के साथ कोड को कॉपी और पेस्ट करना ठीक है, लेकिन हम इसे हाथ से टाइप करने की सलाह देते हैं। यह आपको मांसपेशियों की स्मृति और एक मजबूत समझ विकसित करने में मदद करेगा।
 
-### What Are We Building? {#what-are-we-building}
+### हम क्या बना रहे हैं? {#what-are-we-building}
 
-In this tutorial, we'll show how to build an interactive tic-tac-toe game with React.
+इस ट्यूटोरियल में, हम बताएंगे कि कैसे React के साथ एक इंटरैक्टिव टिक-टैक-टो गेम का निर्माण करना है।
 
-You can see what we'll be building here: **[Final Result](https://codepen.io/gaearon/pen/gWWZgR?editors=0010)**. If the code doesn't make sense to you, or if you are unfamiliar with the code's syntax, don't worry! The goal of this tutorial is to help you understand React and its syntax.
+आप देख सकते हैं कि हम यहां क्या बना रहे हैं: **[अंतिम परिणाम](https://codepen.io/gaearon/pen/gWWZgR?editors=0010)**. यदि कोड आपके लिए मायने नहीं रखता है, या यदि आप कोड के सिंटैक्स से अपरिचित हैं, चिंता मत करें! इस ट्यूटोरियल का लक्ष्य React और इसके सिंटैक्स को समझने में आपकी मदद करना है।
 
-We recommend that you check out the tic-tac-toe game before continuing with the tutorial. One of the features that you'll notice is that there is a numbered list to the right of the game's board. This list gives you a history of all of the moves that have occurred in the game, and is updated as the game progresses.
+हम अनुग्रह करते हैं कि आप ट्यूटोरियल को जारी रखने से पहले टिक-टैक-टो खेल के देखें। सुविधाओं में से एक है कि आप देखेंगे कि खेल के Board के दाईं ओर एक क्रमांकित सूची है। यह सूची आपको खेल में होने वाली सभी चालों का इतिहास देती है, और खेल की प्रगति के रूप में अद्यतन किया जाता है।
```

Comment:
सुविधाओं में से एक है कि आप देखेंगे -> grammar check

Reply:
खेल की प्रगति के रूप में अद्यतन किया जाता है। -> wording + अद्यतन  is too complex

Reply:
> सुविधाओं में से एक है कि आप देखेंगे -> grammar check

How about this 
`आप देख सकते हैं की गेम बोर्ड की दाईं ओर एक क्रमांकित सूची है, ये गेम की कई सुविधाओं में से एक है `

Reply:
> खेल की प्रगति के रूप में अद्यतन किया जाता है। -> wording + अद्यतन is too complex

I can't think of anything else for this. Any suggestions?

Reply:
आप देख सकते हैं की गेम बोर्ड की दाईं ओर एक क्रमांकित सूची है, ये गेम की कई सुविधाओं में से एक है -> perfect

Reply:
और गेम के चलते चलते इसे अपडेट किया जाता है?